docs/templates.md

```diff
@@ -88,4 +88,33 @@ Instead we'd recommend that you ensure that your endpoints perform all I/O,
 for example, strictly evaluate any database queries within the view and
 include the final results in the context.
 
+### A simple example
+
+The first step must be to set the asynchronous rendering mode to enable.
+
+In asynchronous mode you can use 'AsyncTemplateResponse' to render asynchronous functions
+or coroutine objects in jinja2 templates.
+
+```python
+from starlette.templating import Jinja2Templates
+
+templates = Jinja2Templates(directory='templates', enable_async=True)
+
+async def get_name():
```

Comment:
I'd suggest some kind of comment noting that this'd actually only make sense in the case of an actual async operation. (Eg. an async database lookup.)

Reply:
Ok, thanks for your advice.